packages/account/src/Components/sent-email-modal/__tests__/sent-email-modal.spec.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { isMobile } from '@deriv/shared';
+import SendEmailModal from '../sent-email-modal';
```

Comment:
I think here is a typo. The component’s name is SentEmailModal. But we have no error because of default export :)